contracts/governance/src/ZeroExVotes.sol

```diff
@@ -233,18 +245,24 @@ contract ZeroExVotes is IZeroExVotes, Initializable, OwnableUpgradeable, UUPSUpg
             }
         }
 
-        // Leaving here for posterity this is the original OZ implementation which we've replaced
-        // return high == 0 ? 0 : _unsafeAccess(ckpts, high - 1).votes;
-        Checkpoint memory checkpoint = high == 0 ? Checkpoint(0, 0, 0) : _unsafeAccess(ckpts, high - 1);
-        return checkpoint;
+        // TODO: bad. very bad. only works on accident
+        if (high > 0) {
+            success = true;
+            result = _unsafeAccess(ckpts, high - 1);
+        } else {
+            // suppress compiler warning, which really shouldn't be suppressed
+            assembly {
+                result.slot := 0xDEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEF
+            }
+        }
```

Comment:
I am absolutely not suggesting that we make this change to ZeroExVotes. It was simply the most expedient way for me to get my demonstration to work. An actual migration would involve either making this function generic or modifying it in-place.